client/packages/dashboard/src/widgets/StockWidget.tsx

```diff
@@ -13,7 +13,7 @@ import {
 } from '@openmsupply-client/common';
 import { useDashboard } from '../api';
 import { InternalSupplierSearchModal } from '@openmsupply-client/system';
-import { useRequest } from 'packages/requisitions/src/RequestRequisition/api';
+import { useRequest } from '@openmsupply-client/requisitions';
```

Comment:
the imports directly from `packages/...` play havoc with jest and are best avoided.